kong/plugins/jwt/handler.lua

```diff
@@ -6,6 +6,7 @@ local jwt_decoder = require "kong.plugins.jwt.jwt_parser"
 local string_format = string.format
 local ngx_re_gmatch = ngx.re.gmatch
 
+local ngx_var = ngx.var
```

Comment:
I don't think it is a good idea to cache `ngx.var` (a per-request variable) value at the module level... This is why I suggested caching it in the function (as per other parts of our codebase do).

Reply:
I understand. I moved it into the function.